disnake/ext/commands/ctx_menus_core.py

```diff
@@ -30,7 +31,7 @@
 __all__ = ("InvokableUserCommand", "InvokableMessageCommand", "user_command", "message_command")
 
 
-class InvokableUserCommand(InvokableApplicationCommand):
+class InvokableUserCommand(InvokableApplicationCommand, UserCommand):
```

Comment:
What's the point of subclassing `UserCommand`? The chain of `super()` calls never reaches it (see `InvokableApplicationCommand.__init__`) and it doesn't seem to make any typing improvements